changelog

```diff
@@ -15,3 +15,12 @@ inappropriate conduct that falls outside of the scope of the CoC
 - Clarifies scope of applicability, specifically the 'representing the community' clause
 - Adds provisions against personal and political attacks
 - Notes that maintainers are responsible for clarification of questions of acceptable behavior
+
+2.0.0
+- Emphasizes 'community' over 'project' scope. It effectively merges the Community Covenant into the Contributor Covenant
```

Comment:
```suggestion
- Emphasizes 'community' over 'project' scope
```
I think this is sufficient. The additional part breaks the parallelism of the change list.